### ISSUE TITLE
Desktop-Friendly

### ISSUE THREAD

**Issue by lucis** [lucis/alt-zap#8]
Quem diria

**Comment by pedroespindula** [comment]
Rapaz, no Desktop por enquanto ta uma extensão de como é no celular, e acho que ta bem tranquilo. Como tu vê uma melhora na interface?

**Comment by lucis** [comment]
@pedroespindula  No futuro acho que seria legal um design melhor, sem ocupar a tela toda, dividindo em colunas... Mas por enquanto eu acho que já fica ótimo diminuir o container maior e ficar com a aplicação centralizada no meio da tela com um `max-width` de 600px mais ou menos

**Comment by diego-aquino** [comment]
Verdade, @lucis. Acho que ficaria legal dividir em colunas, algo mais ou menos assim...

![Screenshot do display em grid](https://user-images.githubusercontent.com/58959382/91668538-c7df5700-eae3-11ea-89c2-351f113e4e79.png)
